templates/definition/meter/solaredge-hybrid.yaml

```
@@ -66,19 +74,37 @@ render: |
       type: holding
       decode: float32s
   batterymode:
-    source: map
-    values:
-      1: 7 # normal
-      2: 1 # hold
-      3: 3 # charge
+    source: sequence
     set:
-      source: modbus
-      {{- include "modbus" . | indent 4 }}
-      timeout: {{ .timeout }}
-      register:
-        address: 0xE00A # Battery mode
-        type: writesingle
-        decode: uint16
+    - source: const
+      value: 1
```

Eher 60 (s), oder? Der Wert ist ja in Sekunden
```suggestion
      value: 60
```